ci/docker_java_test_all.sh

```diff
@@ -28,6 +28,7 @@ export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
 export ARROW_JAVADOC=1
 bash $SOURCE_DIR/docker_build_java.sh
 
+export ARROW_JAVA_SHADE_FLATBUFS=1
```

Comment:
Does this mean that CI is only going to test with the shaded artifacts?

Reply:
Java 8 is tested unshaded java-11 is tested with shaded.

Reply:
(and I think this only applies to UrsaBot runs).

Reply:
Ah right, ok sounds good